APCAppCore/APCAppCore/DataSubstrate/Model/APCDataMigrationPolicy_Model4ToModel6.m

```diff
@@ -94,7 +93,7 @@ - (NSSet *) linkTasksToV6ScheduleMatchingV4Schedule: (id) scheduleFromV4
         NSFetchRequest *request = [APCTask requestWithPredicate: [NSPredicate predicateWithFormat: @"%K == %@ && %K == %@",
                                                                   NSStringFromSelector (@selector (taskID)),
                                                                   taskIdV6,
-                                                                  NSStringFromSelector (@selector (taskVersionNumber)),
+                                                                  @"taskVersionNumber",
```

Comment:
I'm torn--the string constant is cleaner, and AFAIK always equivalent, but the other way lets Xcode do code completion so you avoid typos. 6 of one, half dozen of the other I guess, but should both instances in this call at least do it the same way, or am I missing something that's different between the two?